packages/vuetify/src/util/helpers.ts

```diff
@@ -30,6 +31,20 @@ export function directiveConfig (binding: BindingConfig, defaults = {}): VNodeDi
   }
 }
 
+export function addEventListener (
```

Comment:
instead of `addEventListener`, `addPassiveEventListener` etc methods I'd rather see something like `eventOptions` and `passiveEventOptions`:

```ts
addEventListener(el, event, cb, options) => el.addEventListener(event, cb, eventOptions(options))
addPassiveEventListener(el, event, cb, options) => el.addEventListener(event, cb, passiveEventOptions(options))
```

(that's not about whether current solutions does what it should do but just about refactoring the structure of current solution)

Reply:
2 questions:

- Where would these live?
- Is this the naming convention we want; instead of `useEventOptions` or something of the sort

Reply:
1. In helpers I think
2. Naming was not important in the comment, rather the idea itself . Use* sounds like effect, probably not a good choice. For passive - maybe makePassive? ¯\_(ツ)_/¯